wbooks/src/app/screens/BookDetail/components/CommentTable/index.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+import {  
+  Text,
+  View,
+  ScrollView
+} from 'react-native';
+
+import CommentItem from './component/CommentItem';
+import styles from './styles';
+
+class CommentTable extends Component {
+
+	render() {
+
+		return (
+      <View style={styles.commentTable}>
+        <ScrollView>
+          <CommentItem/>
+          <CommentItem/>
+          <CommentItem/>
+          <CommentItem/>
+          <CommentItem/>
+        </ScrollView>
```

Comment:
we should be thinking that you will receive a variable comment's list size. Replace this with a FlatList

Reply:
done